packages/forms/docs/10-upgrade-guide.md

```diff
@@ -56,6 +56,10 @@ php artisan vendor:publish --tag=filament-config --force
 rm config/forms.php
 ```
 
+#### Panel configuration is now in each AppPanelProvider
```

Comment:
Belongs in the panels upgrade guide, high impact section. "AppPanelProvider" can just be "panel provider"

Reply:
Ah I see my confusion here. I will edit this.

IMO would useful to have one upgrade page instead of having to chase down multiple, or at least link them to each other. While Filament supports modular usage, structuring the docs around that instead of an "ordinary" use case involving most of the packages makes it confusing to navigate.

Reply:
Yeah, the problem is that people who are not using the panel builder at all will then have to skip past all the stuff that isn't relevant to them